fission-core/src/capabilities/fission.rs

```diff
@@ -211,65 +214,166 @@ mod tests {
     #[test_log::test]
     #[ignore]
     fn test_powerbox_ucan_resource() -> TestResult {
-        let store = InMemoryStore::<RawCodec>::default();
-        let did_verifier_map = DidVerifierMap::default();
+        // let store:  = ucan::delegation::store::MemoryStore::default();
+
+        let server_sk = ed25519_dalek::SigningKey::generate(&mut thread_rng());
+        let server_signer =
+            ucan::did::preset::Signer::Key(ucan::did::key::Signer::EdDsa(server_sk));
+        let server_ed_did_key = EdDidKey::new(server_sk);
```

Comment:
We have incompatible wrappers 😅 Luckily they're all the same key types under the hood

Reply:
Yep. Happy to just adopt your key wrapper. The `EdDidKey` is still an artifact from times where `rs_ucan` didn't have its own wrapper that worked for me.